experiment/simulation/index.html

```diff
@@ -1,13 +1,72 @@
 <!DOCTYPE html>
 <html>
-<head>
-	<!-- Add CSS at the head of HTML file -->
-    <link rel="stylesheet" href="./css/main.css">
-</head>
-<body>
-		<!-- Your code goes here-->
-
-	<!-- Add JS at the bottom of HTML file --> 
-        <script src="./js/main.js"></script>
-</body>
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.1/css/bulma.min.css">
+		<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/virtual-labs/virtual-style@0.0.5-b/css/style.min.css">
+		<link rel="stylesheet" href="./css/main.css">
+	</head>
+
+	<body class="page">
+		<div class="columns is-centered is-flex">
```

Comment:
Why do you need is-flex here? Is that even a valid class?
columns is flex by default

Reply:
Yes, it is a valid class and although columns is flex by default, for some reason it gets overridden for the main workspace and ends up messing up the config for smaller screens.